deploy/03 - MultiFaucet.ts

```diff
@@ -0,0 +1,65 @@
+import {HardhatRuntimeEnvironment} from 'hardhat/types';
+import {DeployFunction} from 'hardhat-deploy/types';
+
+const version = 'v0.1.0';
+const contractName = 'MultiFaucet';
+
+const func: DeployFunction = async (hre: HardhatRuntimeEnvironment) => {
+  async function main() {
+    // Hardhat always runs the compile task when running scripts with its command
+    // line interface.
+    //
+    // If this script is run directly using `node` you may want to call compile
+    // manually to make sure everything is compiled
+    // await hre.run('compile');
+
+    const {deployments, getNamedAccounts, network} = hre;
+    if (!network.live) return;
+    if (!network.tags['staging']) return;
+
+    console.log(`Deploying ${contractName} ${version}`);
+
+    const {deploy} = deployments;
+
+    const {deployer} = await getNamedAccounts();
+
+    const deployResult = await deploy(contractName, {
+      contract: contractName,
+      from: deployer,
+      gasLimit: 2000000,
+      log: true,
+    });
+
+    if (deployResult.newlyDeployed && deployResult.transactionHash) {
+      const blocks = 5;
```

Comment:
Can we refactor this and move it to a function so we can use it in both deploy scripts

Reply:
Yes, but i'll add this in other PR

Reply:
#51